### ISSUE TITLE
Not displaying rating

### ISSUE THREAD

**Issue by Malakay666** [omnight/phonehook#52]
Hi Omnight, please help:
the thing is that Phonehook is running correct, but today I discovered, that one phone number was calling and it displayed "no results". But when I visited server muzutozvednout.cz and inserted phone number there, it has 11 negative ratings. So maybe is there some problem with definition file for CZ. Can you please check and repair? Phone number was +420225027838. Even when I tested in phonehook, there was no results.

And one more thing - is possible to show rating while call comes in format:
11x negative, 4x positive, 6x neutral

it could be handy to decide if is this scum call or not.

**Comment by Malakay666** [comment]
Hi Armin, did you make some changes? I uninstalled czech list and installed again and now it shows rating like I wanted :) and that number is now found :)

**Comment by arminfelder** [comment]
@Malakay666, I did a bit of code clean up with the last release, but there should not have been major changes, but I am glad to read that you are happy now :)

**Comment by Malakay666** [comment]
Yeah, maybe thats why removing and reinserting blocklist helped. I think it stopped working after last update. And then I just tried to remove blocklist and add again and it magically started working again :D
